lemarche/www/siaes/forms.py

```diff
@@ -22,6 +22,16 @@
     ("ZRR", "Zone de revitalisation rurale (ZRR)"),
 )
 
+FORM_CA_CHOICES = (
+    ("", ""),
+    ("-100000", "Moins de 100K€"),
```

Comment:
```suggestion
    (FORM_CA_CHOICES_RANGE_1, "Moins de 100K€"),
```
Je ne sais pas si c'est le nom approprié mais tu peux mettre les keys en variables, ça permet de requête plus facilement et je trouve plus maintenable :)